src/components/QuestionThree.css

```diff
@@ -0,0 +1,33 @@
+/* Style for the container of the dropdown */
+.label {
+    margin: 10px;
+  }
+  
+  /* Style for the dropdown select element */
+  select {
+    width: 100%;
+    padding: 10px;
+    font-size: 16px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    background-color: #fff;
+  }
+  
+  /* Style for the option items in the dropdown */
+  option {
+    font-size: 16px;
+  }
+  
+  /* Media query for smaller screens (adjust as needed) */
+  @media screen and (max-width: 600px) {
```

Comment:
Is there a reason you didn't use the same width as the rest of the app? The drop down still extends outside of the frame when in mobile mode (I used google chromes dev tools and changed mode)

Reply:
Deleted the CSS code. The dropdown issue is currently pending a fix and will be addressed in the upcoming pull request (PR).